examples/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ target_link_libraries(vmi-dump-memory vmi_shared)
 add_executable(vmi-module-list module-list.c)
 target_link_libraries(vmi-module-list vmi_shared)
 
+add_executable(vmi-memory-mapping vmi-memory-mapping.c)
```

Comment:
Not sure if this should be part of the PR? If so, it should be part of the ENABLE_WINDOWS block below

Reply:
Including an example should make testing easier and therefore increase our chances of having this merged quickly.